app/src/main/java/com/example/pictureupload/ui/landing_screen/LandingScreenActivity.kt

```diff
@@ -0,0 +1,30 @@
+package com.example.pictureupload.ui.landing_screen
+
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.material.Surface
+import androidx.compose.material.Text
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import com.example.pictureupload.ui.theme.PictureUploadTheme
+
+class LandingScreenActivity : ComponentActivity() {
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    setContent {
+      PictureUploadTheme {
+        Surface {
+          Box(
+            modifier = Modifier.fillMaxSize(),
+            contentAlignment = Alignment.Center
```

Comment:
We thought it would be better to use multiple activities to keep major features separated e.g. a single activity for the complete authentication flow, a single activity for all the launchScreen composables, and so on. What do you suggest?